src/repository/commands.rs

```diff
@@ -1,4 +1,4 @@
-pub const COMMANDS: [&str; 5] = [
```

Comment:
Question: Do we still need SQL?

Reply:
Let's keep it for integration tests. It provides a working DB implementation that is e2e tested and in RAM. It can be run in CI/CD pipeline in GitHub actions.